tests/ui/conftest.py

```diff
@@ -49,12 +59,58 @@ def jwt_secret(base_url, variables):
 
 
 @pytest.fixture
-def initial_data(transactional_db):
+def initial_data():
     from olympia.amo.tests import addon_factory, user_factory
     from olympia.addons.models import AddonUser
+    from olympia.landfill.collection import generate_collection
+    from olympia.constants.applications import APPS
 
     for x in range(10):
         AddonUser.objects.create(user=user_factory(), addon=addon_factory())
+        generate_collection(addon_factory(), APPS['firefox'])
+
+
+@pytest.fixture
+def gen_addons():
```

Comment:
Please add docstrings for fixtures so it's clear what they're used for. I'm assuming this generates a collection of addons, which means the name of the fixture is misleading. Also, please avoid abbreviations in function names.